README.md

```diff
@@ -144,6 +145,36 @@ $ spanner-cli -p myproject -i myinstance -d mydb -e 'SELECT * FROM users;' -t
 +----+------+--------+
 ```
 
+### Directed reads mode
+
+spanner-cli now supports Directed reads, a feature that allows you to read data from a specific replica of a Spanner database. 
```

Comment:
nit: Maybe we can say `directed reads` instead of `Directed reads` except begging of the line to follow the document: https://cloud.google.com/spanner/docs/directed-reads

Reply:
I've unified the inconsistent.